<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adquisicion</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Ventana">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Adquisicion">
     <attribute name="title">
      <string>Adquisici√≥n</string>
     </attribute>
     <widget class="QScrollArea" name="Inicio">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>821</width>
        <height>581</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Midlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Midlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Midlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="Adquisicion_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>821</width>
         <height>581</height>
        </rect>
       </property>
       <widget class="QLabel" name="logo">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>20</y>
          <width>171</width>
          <height>111</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QListWidget" name="report">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>220</y>
          <width>521</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>470</y>
          <width>521</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="2">
          <widget class="QPushButton" name="exitStar">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: rgb(255, 255, 255);
    selection-background-color: darkgray;
    }

QPushButton:hover {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: rgb(222, 222, 222);
    selection-background-color: darkgray;
    }

QPushButton:pressed {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background:rgb(176, 176, 176);
    selection-background-color: darkgray;
    }
</string>
           </property>
           <property name="text">
            <string>Salir</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="patientData">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: rgb(255, 255, 255);
    selection-background-color: darkgray;
    }

QPushButton:hover {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: rgb(222, 222, 222);
    selection-background-color: darkgray;
    }

QPushButton:pressed {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background:rgb(176, 176, 176);
    selection-background-color: darkgray;
    }
</string>
           </property>
           <property name="text">
            <string>Base de datos pacientes</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="back">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: rgb(255, 255, 255);
    selection-background-color: darkgray;
    }

QPushButton:hover {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: rgb(222, 222, 222);
    selection-background-color: darkgray;
    }

QPushButton:pressed {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background:rgb(176, 176, 176);
    selection-background-color: darkgray;
    }
</string>
           </property>
           <property name="text">
            <string>Atras</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>521</width>
          <height>129</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>                          Tiempo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="time"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="play">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: rgb(255, 255, 255);
    selection-background-color: darkgray;
    }

QPushButton:hover {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: rgb(222, 222, 222);
    selection-background-color: darkgray;
    }

QPushButton:pressed {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background:rgb(176, 176, 176);
    selection-background-color: darkgray;
    }
</string>
           </property>
           <property name="text">
            <string>Iniciar</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QProgressBar" name="alert">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QProgressBar:horizontal {
border: 1px solid gray;
border-radius: 3px;
background: white;
padding: 1px;
text-align: right;
margin-right: 4ex;
}
QProgressBar::chunk:horizontal {
background: qlineargradient(x1: 0, y1: 0.5, x2: 1, y2: 0.5, stop: 0 green, stop: 1 white);
margin-right: 2px; /* space */
width: 10px;
}</string>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QFormLayout" name="formLayout_3"/>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="stop">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: rgb(255, 255, 255);
    selection-background-color: darkgray;
    }

QPushButton:hover {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: rgb(222, 222, 222);
    selection-background-color: darkgray;
    }

QPushButton:pressed {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background:rgb(176, 176, 176);
    selection-background-color: darkgray;
    }
</string>
           </property>
           <property name="text">
            <string>Reiniciar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="formLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>500</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QPushButton" name="adquisitionInfo">
           <property name="font">
            <font>
             <family>Footlight MT Light</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    color: rgb(0, 0, 0);
    border: 2px solid #0e213e;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #1c437d
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #6587ba
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #15325e
        );
    }</string>
           </property>
           <property name="text">
            <string>?</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Visualizacion">
     <attribute name="title">
      <string>Visualizaci√≥n</string>
     </attribute>
     <widget class="QScrollArea" name="Inicio_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>821</width>
        <height>561</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Midlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Midlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Midlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>821</width>
         <height>561</height>
        </rect>
       </property>
       <widget class="QPushButton" name="adquisitionInfo2">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>490</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Footlight MT Light</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    color: rgb(0, 0, 0);
    border: 2px solid #0e213e;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #1c437d
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #6587ba
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #15325e
        );
    }</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
       </widget>
       <widget class="QLabel" name="logo2">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>0</y>
          <width>171</width>
          <height>111</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QListView" name="listView">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>120</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>481</width>
          <height>612</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="PlotWidget" name="viewSignalFCz">
           <property name="backgroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
           <property name="foregroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PlotWidget" name="viewSignalOz">
           <property name="backgroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
           <property name="foregroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PlotWidget" name="viewSignalO1">
           <property name="backgroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
           <property name="foregroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PlotWidget" name="viewSignalPO7">
           <property name="backgroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
           <property name="foregroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PlotWidget" name="viewSignalO2">
           <property name="backgroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
           <property name="foregroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PlotWidget" name="viewSignalPO8">
           <property name="backgroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
           <property name="foregroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PlotWidget" name="viewSignalPO4">
           <property name="backgroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
           <property name="foregroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PlotWidget" name="viewSignalPO3">
           <property name="backgroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
           <property name="foregroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
